### PR TITLE
Fix with run tests and improvements pep8

### PR DESCRIPTION
**Pull request by mapeveri**

### CODE REVIEW

File: masonite/session.py [hunk 1]
```diff
@@ -36,44 +34,39 @@ def set(self, key, value):
...
 
-
     def flash(self, key, value):
         """
         Add temporary data to the session
         """
 
         ip = self.__get_client_address()
-        if not ip in self._flash:
+        if not (ip in self._flash):
```
Comment (josephmancuso) [inline masonite/session.py]: Is this really PEP 8? It looks like a useless case of parenthesis. 

(ip in self._flash) is setting a boolean value (True or False) and you are "not"ing it.

maybe we should change it to `if ip not in self._flash` to make it more "Pythonic" (I hate that word)

File: requirements.txt [hunk 2]
```diff
@@ -10,4 +10,5 @@ requests==2.18.4
 boto3==1.5.24
 tldextract==2.2.0
 pusher==1.7.4
-ably==1.0.1
\ No newline at end of file
+ably==1.0.1
+pendulum==1.4.4
```
Comment (josephmancuso) [inline requirements.txt]: good catch here 👍

File: setup.py [hunk 3]
```diff
@@ -35,7 +35,8 @@
         'cryptography==2.1.4',
         'PyMySQL==0.8.0',
         'psycopg2==2.7.3.2',
-        'masonite-entry>=0.0.0,<=0.9.99'
+        'masonite-entry>=0.0.0,<=0.9.99',
+        'pendulum==1.4.4',
     ],
     description='The core for the Masonite',
```
Comment (josephmancuso) [inline setup.py]: Can we actually change this to `description='The core for the Masonite framework',`

File: masonite/session.py (hunk 1 above)
Comment (mapeveri) [reply masonite/session.py]: Yes, if ip not in self._flash it's better :)

File: setup.py (hunk 3 above)
Comment (mapeveri) [reply setup.py]: Perfect

**Comment by mapeveri** [comment]
Ready :)

**Pull request merged**